lib/provider.coffee

```diff
@@ -22,21 +27,32 @@ class BibtexProvider
   second `@`, as this would become confusing.
   ###
   wordRegex: XRegExp('(?:^|[\\p{WhiteSpace}\\p{Punctuation}])@[\\p{Letter}\\p{Number}\._-]*')
-  constructor: ->
-    @bibtex = []
+  atom.deserializers.add(this)
```

Comment:
I'm going to go ahead and merge this and move these definitions in my next commit.